src/sagas.js

```diff
@@ -8,7 +8,7 @@ import { all } from 'redux-saga/effects';
 
 import { appInit } from 'components/App/AppSagas';
 import { loginInit } from 'components/LoginPage/LoginPageSagas';
-import { logoutButtonInit } from 'components/LogoutButton/LogoutButtonSagas';
+import { logoutInit } from 'components/NotesPage/NotesPageTopBar/NotesPageTopBarSagas';
```

Comment:
We name sagas by the component name, e.g.:
`NoteDetailsSagas` -> `noteDetailsInit`
`NotesPageTopBarSagas` -> `notesPageTopBarInit`